dvc/repo/experiments/show.py

```diff
@@ -45,20 +45,22 @@ def _collect_experiment_commit(
             res["params"] = params
 
         res["deps"] = {
-            dep.def_path: {
+            relpath(dep.fs_path): {
```

Comment:
Fixes https://github.com/iterative/dvc/issues/7575#issuecomment-1134242482

Reply:
I think this should be relative to the repo root and not cwd? Otherwise the output will change depending on where you run `exp show` in the repo

Reply:
Updated